libpod/container.go

```diff
@@ -365,6 +366,12 @@ type ContainerConfig struct {
 
 	// Systemd tells libpod to setup the container in systemd mode
 	Systemd bool `json:"systemd"`
+
+	// HasHealthCheck tells us if the container has a health check it should
+	// be running
+	HasHealtchCheck bool `json:"hasHealthCheck"`
```

Comment:
Spelling?

Reply:
Actually, can we just get rid of this entirely? If HealthCheckConfig is nil, that means no health check, right?